dev/sbt-plugin/src/sbt-test/sbt-plugin/distribution/build.sbt

```diff
@@ -53,7 +53,8 @@ InputKey[Unit]("absence") := {
 }
 
 lazy val checkDevRuntimeClasspath = inputKey[Unit]("Checks presence of a jar in the DevRuntime classpath")
-checkDevRuntimeClasspath <<= checkDevClasspathTask(Internal.Configs.DevRuntime)
+
+checkDevRuntimeClasspath := checkDevClasspathTask(Internal.Configs.DevRuntime)
```

Comment:
You can run the test locally from the `sbt` console by running `scripted sbt-plugin/distribution`.